db/schema.rb

```diff
@@ -20,16 +20,16 @@
     t.string   "name",              null: false
     t.boolean  "is_paid"
     t.boolean  "is_permanent_paid"
-    t.datetime "created_at"
-    t.datetime "updated_at"
+    t.datetime "created_at",        null: false
```

Comment:
@Skud do you think this should block merging the PR? Otherwise, it looks good to me.